inst/uvhydrograph/uvhydrograph.Rmd

```diff
@@ -38,7 +38,10 @@ cat(getLogo())
 </div>
 
 <div class="header">
-`r if(!isEmptyOrBlank(fetchReportMetadataField(data, 'excludeZeroNegative')) || !isEmptyOrBlank(fetchReportMetadataField(data, 'excludeCorrections')) || !isEmptyOrBlank(fetchReportMetadataField(data, 'excludeDiscrete'))) {paste("**Advanced Options:**") }` `r if(!isEmptyOrBlank(fetchReportMetadataField(data, 'excludeZeroNegative'))) { paste("Zero/negative values excluded. ") }` `r if(!isEmptyOrBlank(fetchReportMetadataField(data, 'excludeCorrections'))) { paste("Delete region corrections excluded.") }` `r if(!isEmptyOrBlank(fetchReportMetadataField(data, 'excludeDiscrete'))) { paste("Discrete data excluded.") }`
+`r if(fetchReportMetadataField(data, 'excludeZeroNegative') || !isEmptyOrBlank(fetchReportMetadataField(data, 'excludeCorrections')) || fetchReportMetadataField(data, 'excludeDiscrete')) {paste("**Advanced Options:**") }` 
```

Comment:
cleaned this up a bit, before the zeroNeg and discrete params only showed up when they were true, now they show up with false, so had to change the logic slightly.